### ISSUE TITLE
Incorrect heading structure

### ISSUE THREAD

**Issue by iadawn** [w3c/w3c-website#276]
Location: [Group participants page](https://beta.w3.org/groups/wg/eowg/participants/)

The participants page includes headings for the different participant groups and each participant name is also a heading. They are all marked up as `<h2>` so the visual heirarchy is not communicated correctly.

<img width="585" alt="sample of heading structure" src="https://user-images.githubusercontent.com/2444840/223711008-66e709fa-7ee9-4f9c-8789-b24a0c8b0d7e.png">

Easy solution is to change the heading level of the participant names to be `<h3>`.

**Comment by deniak** [comment]
The heading should be fixed on the different pages listing users.
